docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -372,6 +372,16 @@ curl 'https://<kibana_url>/api/fleet/agents/<agentID>/unenroll' \
   --compressed
 ----
 
+Another possible solution to fix this without force unenroll is to use the 'dev tools' to force the stale {agent} to upgrade:
+
+[source,shell]
+----
+POST kbn:/api/fleet/agents/5f18484c-17d5-12fe-c455-fcde50f8b/upgrade
+{ "force": true, "version": "8.7.0" }
+----
+
+In this example, we are upgrading the agent with agent-id *5f18484c-17d5-12fe-c455-fcde50f8b* to version *8.7.0*. The dev tools can be found by browsing to Management -> Dev Tools of your instance. The agent-id of an Agent can be found by browsing to Management -> Fleet -> Search for the Agent based, for example, hostname.
```

Comment:
```suggestion
Another possible solution to fix this problem without using a force unenroll is to use the {kib} **Dev Tools** to force the stale {agent} to upgrade:

[source,shell]
----
POST kbn:/api/fleet/agents/5f18484c-17d5-12fe-c455-fcde50f8b/upgrade
{ "force": true, "version": "8.7.0" }
----

This example upgrades the agent with agent-id `5f18484c-17d5-12fe-c455-fcde50f8b` to version `8.7.0`. The Dev Tools can be found by browsing in {kib} to **Management -> Dev Tools**. The `agent-id` of an agent can be found by browsing to **Management -> Fleet** and then searching for the agent based, for example, on the hostname where the agent is installed.
```
Here's some small suggested changes. I'm not sure if the last line is technically correct, but I was sure what `based` meant. Hopefully I guessed correctly. :-)